sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpTransactionEnlistment.cs

```diff
@@ -61,7 +60,7 @@ void IPromotableSinglePhaseNotification.Initialize()
         void IPromotableSinglePhaseNotification.SinglePhaseCommit(SinglePhaseEnlistment singlePhaseEnlistment)
         {
             _transactionManager.RemoveEnlistment(_transactionId);
-            TaskExtensionHelper.Schedule(() => SinglePhaseCommitAsync(singlePhaseEnlistment));
+            _ = SinglePhaseCommitAsync(singlePhaseEnlistment);
```

Comment:
So the way that the Transactions work is using the TransactionScope. Since the methods on TransactionScope are not async, even though we are doing async operations when a user commits/rolls back, the user is not able to await the transaction operation. This isn't ideal, but you can argue that if there is something that they want to do only after the transaction is complete, they can probably include it in the transaction, provided it is on the same entity.